core/src/main/kotlin/com/gatehill/corebot/chat/model/template/CustomActionTemplate.kt

```diff
@@ -18,7 +18,7 @@ abstract class CustomActionTemplate : BaseActionTemplate() {
             val options = transform(actionConfig, placeholderValues)
             CustomAction(actionType,
                     buildShortDescription(actionConfig),
-                    if (actionMessageMode == ActionMessageMode.INDIVIDUAL) buildStartMessage(options, actionConfig) else null,
+                    if (actionMessageMode == ActionMessageMode.INDIVIDUAL && actionConfig.showJobOutcome) buildStartMessage(options, actionConfig) else null,
```

Comment:
So users can opt out of showing those messages as the reactions already give that message. Reducing the output for simple jobs